src/lib.rs

```diff
@@ -49,46 +50,33 @@
 #![doc(html_logo_url = "https://async.rs/images/logo--hero.svg")]
 #![recursion_limit = "2048"]
 
-/// Declares stable items.
-#[doc(hidden)]
-macro_rules! cfg_stable {
-    ($($item:item)*) => {
-        $(
-            #[cfg(feature = "stable")]
-            $item
-        )*
-    }
-}
-
-cfg_stable! {
-    #[macro_use]
-    mod utils;
+#[macro_use]
+mod utils;
 
-    pub mod fs;
-    pub mod future;
-    pub mod io;
-    pub mod net;
-    pub mod os;
-    pub mod path;
-    pub mod prelude;
-    pub mod stream;
-    pub mod sync;
-    pub mod task;
+pub mod fs;
```

Comment:
I think we may want to keep this behind the `default` flag. If we change the behavior of passing `no-default` that should be considered a breaking change.

Reply:
The whole library is now behind the `default` flag. (or perhaps I misunderstood your point?)